csharp/Glacier2/Callback/README.md

```diff
@@ -17,33 +17,33 @@ Follow these steps to build and run the demo:
 
 1. Build the client and server applications:
 
-``` shell
-dotnet build
-```
```

Comment:
Do we need to indent code blocks like this?

Reply:
Yes, lists in markdown cannot have other bits in between them that are not indented. This makes markdown parsers and formatters very unhappy.

```markdown
1. item a
this is bad
2. item b
   this is okay
3. item c
```

Reply:
All the C# ones need fixing. I'll do in a separate PR.